src/towncrier/test/test_build.py

```diff
@@ -1769,3 +1769,54 @@ def test_no_ignore_configured(self, runner):
             _main, ["--draft", "--date", "01-01-2001", "--version", "1.0.0"]
         )
         self.assertEqual(0, result.exit_code, result.output)
+
+    @with_project(
+        config="""
+    [tool.towncrier]
+    title_format = "{version} - {project_date}"
+
+      [[tool.towncrier.type]]
+      directory = "feature"
+      name = "Feature"
+    """
+    )
+    def test_showcontent_default_toml_array(self, runner):
+        """
+        When configuring custom fragment types with a TOML array
+        `showcontent` should default to `true`.
```

Comment:
Very minor comment https://jml.io/test-docstrings/

```suggestion
        When configuring custom fragment types with a TOML array
        a missing `showcontent` defaults to `true`.
```